ui-common/src/main/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -933,7 +944,32 @@ private void loginToGoogleIfNeeded(String s) {
         }
 
         waitForCallbackRedirect("google");
-        fillAndValidateGoogleAccount(s);
+
+        Account account = AccountsDirectory.getInstance().get(s);
+
+        // change language if needs
+        if (!WebDriverRunner.getWebDriver().getCurrentUrl().contains("&hl=en&")) {
```

Comment:
When the correct language is already set, the element is not clickable. So we need to check before whether the incorrect language is set.